src/components/App.jsx

```diff
@@ -50,7 +49,6 @@ const App = () => {
           </Content>
         </Main>
         <Alerter t={t} />
-        <IconSprite />
```

Comment:
Doesn't the cozy-bar need the sprite? Remember the bar in production is on v7. Can you check it?

Reply:
Dans le package.json on utilise la v7, donc il ne devrait pas y avoir de différence si ? (désolé de te faire répéter). J'ai souvenir qu'en prod on utilise une cozy-bar servie par la stack, mais a priori il ne devrait pas y avoir de différence là si ?

> Can you check it?
C'est à dire, checker quoi ? De déployer en prod une version de test voir ce qu'il se passe ?

Reply:
Il me semble qu'on a des boutons par exemple https://github.com/cozy/cozy-bar/commit/3a2031a6199d06bfe94680224c2e5a01b3bedf3c#diff-f9f5ca220b88bfb8f8035277c7bce0dbd02c114a1f4596a92ef53726ce0de509L57

qui utilisent encore des noms en v7 puisqu'ici ce changement est livré à partir de la V8. 

Donc je continue de penser que le fait de retirer le sprite casse des choses.

Reply:
ah checker si la v7 se base bien sur les sprites... Ok merci :)